dali/imgcodec/decoders/nvjpeg/nvjpeg_test.cc

```diff
@@ -72,6 +72,32 @@ class NvJpegDecoderTest : public NumpyDecoderTestBase<GPUBackend, OutputType> {
     return std::make_shared<JpegParser>();
   }
 
+  /**
+   * @brief Checks if the image is similar to the reference.
+   *
+   * Checks if 1) max error is lower than 30% 2) mean squared error is lower than 1%.
```

Comment:
The implementation has mean absolute value (L1 norm), not mean square (L2 norm).

Reply:
Oh, I forgot to change the comment. But I'll make it mean square in a moment as suggested

Reply:
Changed. Thanks for the catch